apps/trading/components/amm/liquidity-form.tsx

```diff
@@ -0,0 +1,402 @@
+import { zodResolver } from '@hookform/resolvers/zod';
+import { BarChart3Icon } from 'lucide-react';
+import { useState } from 'react';
+import { useForm, useFormState } from 'react-hook-form';
+import { z } from 'zod';
+import { Button } from '../ui/button';
```

Comment:
It can be a follow up task. But for some of the ui components lets switch to ui-toolkit.